cmd/zc_pipeline_init.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/Azure/azure-storage-azcopy/v10/common"
 )
 
-func InitPipeline(ctx context.Context, location common.Location, credential common.CredentialInfo, logLevel pipeline.LogLevel, trailingDot common.TrailingDotOption) (p pipeline.Pipeline, err error) {
+func InitPipeline(ctx context.Context, location common.Location, credential common.CredentialInfo, logLevel pipeline.LogLevel, trailingDot common.TrailingDotOption, from common.Location) (p pipeline.Pipeline, err error) {
```

Comment:
Nit: Considering we are already passing 'To' we could just send in entire FromTo() and reduce number of args

Reply:
This causes some issues when the location is Local and such and there is one spot where the location isn't passed as a FromTo. Regardless, this code will be removed with the migration, so shouldnt matter if we keep 1 vs 2 params here.